### ISSUE TITLE
Placeholder type specifiers will have their return type duplicated. 

### ISSUE THREAD

**Issue by wroyca** [marzer/poxy#26]
## Environment

**version and/or commit hash:** f4607c4e3a6b35fcace92f6858fb663c3863037c

## Describe the bug

Placeholder type specifiers will have their return type duplicated, for example:

```cpp
auto 
hello ()
{
  return 5;
}
```

will be documented as:

```cpp
auto
hello () -> auto
{
  return 5;
}
```

## Additional information

We've already addressed scenarios involving trailing return types de-duplication, so it appears that this is merely a minor oversight

**Comment by marzer** [comment]
Ah yeah that's an easy fix, the same workaround I used for the trailing types just needs a tweak

**Comment by marzer** [comment]
fixed in v0.13.0 :)
